WordPress/gutenberg
# DataViews: Restore focus style in Grid view

Related to #64209

## What?

This PR restores the outline when the preview button is focused in the grid layout.

![image](https://github.com/user-attachments/assets/fd0f4a92-0b3f-434a-b887-4041ce6897ae)

## Why?

This is because `overflow: hidden` is applied to the `.dataviews-view-grid__media` element that wraps the button.

## How?

According to [this comment](https://github.com/WordPress/gutenberg/pull/64209#discussion_r1703803664), it seems that this is to prevent the preview from going outside the container, but since the button component itself has `overflow: hidden`, I was not able to reproduce the problem in my testing.

Template:
https://github.com/WordPress/gutenberg/blob/f12b78b7c4e830032d7f9a4641df40236644683b/packages/edit-site/src/components/page-templates/style.scss#L14

Page:

https://github.com/WordPress/gutenberg/blob/f12b78b7c4e830032d7f9a4641df40236644683b/packages/edit-site/src/components/post-list/style.scss#L43

Patterns:
https://github.com/WordPress/gutenberg/blob/f12b78b7c4e830032d7f9a4641df40236644683b/packages/edit-site/src/components/page-patterns/style.scss#L27


## Testing Instructions

### Testing Instructions for Keyboard

- Visit the Pages, Templates, and Patterns grid layout.
- Use the tab key to focus the preview button.
- You should see an outline.
